### PR TITLE
cam_cesm2_2_rel_01: CAM-Chem CONUS ref cases; fv3 abort if non-hydrostatic

### LINKED ISSUE #172: Ref cases for CAM-Chem compsets on SE CONUS refined grid

### LINKED ISSUE #177: Add endrun to prevent non-hydrostatic FV3 configurations
Once the CESM2.2 branch is made, add an endrun to prevent non-hydrostatic FV3 configurations from running

### LINKED ISSUE #206: CAM_ROOT not correctly set in test_driver.sh
Replace line 105 with:
export CAM_ROOT="$( dirname $( dirname $( cd ${test_dir}; pwd -P ) ) )"

### PR DESCRIPTION
**Pull request by fvitt**
modified:   cime_config/config_compsets.xml
        modified:   src/dynamics/fv3/dyn_comp.F90

closes #172 
closes #177 
fixes #206

### CODE REVIEW

File: cime_config/config_compsets.xml [hunk 1]
```diff
@@ -640,56 +640,29 @@
...
-<!--    </entry> -->
+    <entry id="RUN_TYPE">
+       <values match="first">
+	<value compset="HIST_CAM60%CCTS1" grid="a%ne0np4CONUS">hybrid</value>
+      </values>
+    </entry>
+
+    <entry id="RUN_REFCASE">
+      <values match="first">
+	<value compset="HIST_CAM60%CCTS1" grid="a%ne0np4CONUS">f.e22.FCnudged.ne0CONUSne30x8_ne0CONUSne30x8_mt12.cesm2_2_alpha06b.001</value>
```
Comment (cacraigucar) [inline cime_config/config_compsets.xml:651]: I just want to confirm that this change will apply to the following compsets (FCSD was not mentioned in last night's email discussion):
FCHIST
FCNudged
FCSD

Also, I assume the refcases need to be stored in inputdata, so double checking that this has been done if needed.

Comment (fvitt) [reply cime_config/config_compsets.xml:651]: > I just want to confirm that this change will apply to the following compsets (FCSD was not mentioned in last night's email discussion):
> FCHIST
> FCNudged
> FCSD
> 
> Also, I assume the refcases need to be stored in inputdata, so double checking that this has been done if needed.

Arg.  This should not apply to FCSD.

**Review by gold2718 — APPROVED** [review]
This looks okay -- after you clear up the FCSD bit.
Is it as simple as changing the match to:
`HIST_CAM60%CCTS1.*_CLM50`?

**Review by jtruesdal — APPROVED** [review]
I can only speak to the fv3 mods but they are correct.

**Review by nusbaume — APPROVED** [review]
Looks good, thanks!

**Pull request merged**